Tools/TestWebKitAPI/Tests/WebCore/LayoutUnitTests.cpp

```diff
@@ -277,13 +277,15 @@ TEST(WebCoreLayoutUnit, FromFloatCeil)
     ASSERT_EQ(LayoutUnit(1.25f + tolerance), LayoutUnit::fromFloatCeil(1.25f + tolerance / 2));
     ASSERT_EQ(LayoutUnit(), LayoutUnit::fromFloatCeil(-tolerance / 2));
 
+#if !COMPILER(MSVC)
```

Comment:
These sections cause warnings about overflows, and warnings are treated as errors.